Modix.Data/Models/Moderation/ModerationActionTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Modix.Data.Models.Moderation
+{
+    public enum ModerationActionTypes
+    {
+        InfractionCreated,
```

Comment:
Non-flag enums should be singular.

Reply:
Are these stored in the DB as numbers? My intention would be for them to be stored as strings, to avoid this specific problem. Or, that we have a key-name table in the DB.

Reply:
An enum is an int, you should save it as such to the database.

Reply:
and yet, like you say, saving it as an `int` allows for ambiguity, unless the database also contains a table containing the name/id mapping. In my mind, an ideal database should be fully self-descriptive, where understanding the data doesn't require any context from higher layers.

Of course, this is a moot point if postgres doesn't actually allow for mapping `enum`'s by name.